core/src/main/scala/scalacss/Css.scala

```diff
@@ -2,9 +2,12 @@ package scalacss
 
 object Css {
 
-  def apply(styles: TraversableOnce[StyleA])(implicit env: Env): Css =
+  def prepareStyles(styles: TraversableOnce[StyleA])(implicit env: Env): Stream[CssStyleEntry] =
```

Comment:
Instead of `Stream[CssStyleEntry]` could you pls create an alias (like there was `Css` before).